inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/postgres-cdc/src/main/java/org/apache/inlong/sort/cdc/postgres/table/PostgreSQLReadableMetaData.java

```diff
@@ -467,6 +478,35 @@ private static List<String> getPkNames(@Nullable TableChange tableSchema) {
         return tableSchema.getTable().primaryKeyColumnNames();
     }
 
+    /**
+     * get a map about column name and type
+     * @param tableSchema
+     * @return map of field name and field type
+     */
+    private static Map<String, String> getPostGreType(@Nullable TableChanges.TableChange tableSchema) {
+        if (tableSchema == null) {
+            return null;
+        }
```

Comment:
The `getPostGreType` method has similar implementations in MySQL CDC(`getMysqlType`) and Oracle CDC (`getOracleType`). Can we extract them and put them in a util class?

Reply:
sure thx